README.rst

```diff
@@ -77,24 +28,27 @@ Citation
 
 When you use the BluePyOpt software or method for your research, we ask you to cite the following publication (**this includes poster presentations**):
 
-[Van Geit W, Gevaert M, Chindemi G, Rössert C, Courcol J, Muller EB, Schürmann F, Segev I and Markram H (2016). BluePyOpt: Leveraging open source software and cloud infrastructure to optimise model parameters in neuroscience. Front. Neuroinform. 10:17. doi: 10.3389/fninf.2016.00017](http://journal.frontiersin.org/article/10.3389/fninf.2016.00017)
-
-```bibtex
-@ARTICLE{bluepyopt,
- AUTHOR={Van Geit, Werner  and  Gevaert, Michael  and  Chindemi, Giuseppe  and  Rössert, Christian  and  Courcol, Jean-Denis  and  Muller, Eilif Benjamin  and  Schürmann, Felix  and  Segev, Idan  and  Markram, Henry},   
-TITLE={BluePyOpt: Leveraging open source software and cloud infrastructure to optimise model parameters in neuroscience},
-JOURNAL={Frontiers in Neuroinformatics},
-VOLUME={10},
-YEAR={2016},
-NUMBER={17},
-URL={http://www.frontiersin.org/neuroinformatics/10.3389/fninf.2016.00017/abstract},
-DOI={10.3389/fninf.2016.00017},
-ISSN={1662-5196}
-}
-```
+`Van Geit W, Gevaert M, Chindemi G, Rössert C, Courcol J, Muller EB, Schürmann F, Segev I and Markram H (2016). BluePyOpt: Leveraging open source software and cloud infrastructure to optimise model parameters in neuroscience. Front. Neuroinform. 10:17. doi: 10.3389/fninf.2016.00017 <http://journal.frontiersin.org/article/10.3389/fninf.2016.00017>`_.
+
+.. code-block:: 
+
+	bibtex
```

Comment:
Please remove the 'bibtex' here. Not sure why this had to be changed?

Reply:
I did not make any change, but I can remove it

Reply:
Didn't you switch to 'code-block' instead of ```bibtex ?